### DISCUSSION TITLE
JDK 17 support

### DISCUSSION THREAD

**Discussion by henrychess** [redhat-developer/vscode-java#2140] [General]
Hello! I've just installed JDK 17 as well as this extension in VS Code, but it keeps saying that it doesn't support JDK 17. But here it says last update is 9 days ago and it has added support for JDK 17. Is there a way for me to install this update on my computer so that I don't have to downgrade to JDK 11?

**Comment by snjeza** [comment]
@henrychess You can try https://download.jboss.org/jbosstools/jdt.ls/staging/java-0.83.0-388.vsix

**Comment by fbricon** [comment]
vscode-java 1.0.0 with Java 17 support has been released in the VS Code and Open VSX marketplaces.
